### PR TITLE
improve issue templates

### LINKED ISSUE #79110: issue template improvements
- [ ] 1. separate `<details>` logs section to each command log

<details>
  <summary>instead of this</summary>

```html
<details>
  <summary>Logs</summary>

<!--
      Run your application with `flutter run --verbose` and attach all the
      log output below between the lines with the backticks. If there is an
      exception, please see if the error message includes enough information
      to explain how to solve the issue.
-->

    ```
    ```

<!--
     Run `flutter analyze` and attach any output of that command below.
     If there are any analysis errors, try resolving them before filing this issue.
-->

    ```
    ```

<!-- Finally, paste the output of running `flutter doctor -v` here. -->

    ```
    ```

</details>
```
</details>

<details>
  <summary>i think it would be better to have this</summary>

```html
<details>
  <summary>flutter run --verbose</summary>

<!--
      Run your application with `flutter run --verbose` and attach all the
      log output below between the lines with the backticks. If there is an
      exception, please see if the error message includes enough information
      to explain how to solve the issue.
-->

    ```
    ```
</details>


<details>
  <summary>flutter analyze</summary>

<!--
     Run `flutter analyze` and attach any output of that command below.
     If there are any analysis errors, try resolving them before filing this issue.
-->

    ```
    ```
</details>


<details>
  <summary>flutter doctor -v</summary>

  <!-- Finally, paste the output of running `flutter doctor -v` here. -->

    ```
    ```
</details>
```
</details>

- [ ] 2. include a requirement of minimal reproducible sample and give simple instructions how to make it. point out that without ability to reproduce the issue the team will very likely close it. i'm actually surprised it's not there already, because most of the issue require it. this would look like this

```html
<details>
<summary>code sample</summary>

    <!-- some intructions -->

    ```
    ```
</details>
```

- [ ] 3. similarly to the above, include a section for attaching screenshots with `<details>`
- [ ] 4. include a section suggesting attaching videos. suggest directly uploading it on github as ***last*** entries in the issue, warn about platform 10mb size restriction
- [ ] 5. make a template like "Let me in" for people who don't need all these comments and already know exactly what they need to attach, this would include only an empty  `<details>` for easy copy-paste

### PR DESCRIPTION
**Pull request by nt4f04uNd**
fixes https://github.com/flutter/flutter/issues/79110

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_activation.md [hunk 1]
```diff
@@ -17,34 +17,45 @@ assignees: ''
...
-<!-- Please tell us exactly how to reproduce the problem you are running into. -->
+<!--
+     Please describe in detail the problem you are running into.
+
+     If you have concrete steps to reproduce the problem, list them:
+
+        1. ...
+        2. ...
+        3. ...
+-->
```
Comment (Hixie) [inline .github/ISSUE_TEMPLATE/1_activation.md:36]: does this run the risk of people filling in the "..." parts not realising it's commented out?

File: .github/ISSUE_TEMPLATE/1_activation.md [hunk 2]
```diff
@@ -17,34 +17,45 @@ assignees: ''
...
 
 <!--
-      Include the full logs of the commands you are running between the lines
-      with the backticks below. If you are running any "flutter" commands,
-      please include the output of running them with "--verbose"; for example,
-      the output of running "flutter --verbose create foo".
+     Include the full logs of the commands you are running between the lines
+     with the backticks below. If you are running any "flutter" commands,
+     please include the output of running them with "-v"; for example,
+     the output of running "flutter -v create foo".
```
Comment (Hixie) [inline .github/ISSUE_TEMPLATE/1_activation.md:47]: why not --verbose?

File: .github/ISSUE_TEMPLATE/2_bug.md [hunk 3]
```diff
@@ -16,36 +16,75 @@ assignees: ''
...
+<!-- What did you want to see? -->
 
-<!-- You must include full steps to reproduce so that we can reproduce the problem. -->
+### Actual results
+
+<!-- What did you see? -->
 
-1. Run `flutter create bug`.
-2. Update the files as follows: ... <!-- include every file that is different from the template app! -->
-3. ... <!-- describe how to reproduce the problem -->
```
Comment (Hixie) [inline .github/ISSUE_TEMPLATE/2_bug.md:30]: we seem to have lost the `flutter create ...` step here

File: .github/ISSUE_TEMPLATE/7_custom.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<details>
+<summary></summary>
+
+</details>
```
Comment (Hixie) [inline .github/ISSUE_TEMPLATE/7_custom.md:14]: we don't need this, people can just use the "blank issue" link

File: .github/ISSUE_TEMPLATE/5_performance_speed.md [hunk 5]
```diff
@@ -60,10 +60,31 @@ assignees: ''
...
+
+<details>
+<summary>Code sample</summary>
+
+<!--
+     Please create a minimal reproducible sample that shows the problem
+     and attach it below between the lines with the backticks.
+
+     Without this we will unlikely progress on the issue, and thus will have
+     to close it.
```
Comment (Hixie) [inline .github/ISSUE_TEMPLATE/5_performance_speed.md:73]: this should probably be phrased in a way that says we're unlikely to be _able_ to progress on the issue, and that we will _regretfully_ have to close it, or some such.

File: .github/ISSUE_TEMPLATE/5_performance_speed.md [hunk 6]
```diff
@@ -16,35 +16,35 @@ assignees: ''
...
      Please read our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
 ## Details
 
 <!--
-1.  Please tell us exactly how to reproduce the problem you are running into.
-
-2.  Please attach a small application (ideally just one main.dart file) that
-     reproduces the problem. You could use https://gist.github.com/ for this.
```
Comment (Hixie) [inline .github/ISSUE_TEMPLATE/5_performance_speed.md:29]: we seem to have lost #2 here?

Comment (Hixie) [reply .github/ISSUE_TEMPLATE/5_performance_speed.md:29]: oh i see you moved it below. I really think this is the most important point though. Also the next paragraph says "this app" which makes no sense if we move #2 down.

File: .github/ISSUE_TEMPLATE/1_activation.md (hunk 2 above)
Comment (nt4f04uNd) [reply .github/ISSUE_TEMPLATE/1_activation.md:47]: to line this up with other commands that suggest using `-v`. and why to ask verbosily type `--verbose`?

File: .github/ISSUE_TEMPLATE/1_activation.md (hunk 1 above)
Comment (nt4f04uNd) [reply .github/ISSUE_TEMPLATE/1_activation.md:36]: it does, will bring it back out of comment

File: .github/ISSUE_TEMPLATE/7_custom.md (hunk 4 above)
Comment (nt4f04uNd) [reply .github/ISSUE_TEMPLATE/7_custom.md:14]: ok

File: .github/ISSUE_TEMPLATE/2_bug.md (hunk 3 above)
Comment (nt4f04uNd) [reply .github/ISSUE_TEMPLATE/2_bug.md:30]: such meta-form of reproduction steps doesn't bring anything productive to triging, it makes people think they should verbosily list every step they made to ***create a reproduction*** rather than describe actual steps to reproduce

i moved the requirement of the reproducible code sample below in the `<details>`

if people come to this issue tracker they definitely know how the create a flutter app, so i excluded this note as not needed

File: .github/ISSUE_TEMPLATE/2_bug.md [hunk 7]
```diff
@@ -16,36 +16,75 @@ assignees: ''
...
+### Description
+
+<!--
+     Please describe in detail the problem you are running into.
+
+     List concrete steps to reproduce the problem, if this will better illustrate it:
+
+        1. ...
+        2. ...
+        3. ...
```
Comment (nt4f04uNd) [inline .github/ISSUE_TEMPLATE/2_bug.md:35]: @Hixie does it make sense to move it out of comment too (as above), or rather remove these lines at all?

**Comment by nt4f04uNd** [comment]
ready for another review

File: .github/ISSUE_TEMPLATE/5_performance_speed.md (hunk 6 above)
Comment (nt4f04uNd) [reply .github/ISSUE_TEMPLATE/5_performance_speed.md:29]: moved related sections down below

File: .github/ISSUE_TEMPLATE/2_bug.md [hunk 8]
```diff
@@ -16,36 +16,72 @@ assignees: ''
...
 
+     Issues directly asking for support will be closed.
+
      If you have found a bug or if our documentation doesn't have an answer
      to what you're looking for, then fill out the template below. Please read
      our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
-## Steps to Reproduce
+## Description
```
Comment (nt4f04uNd) [inline .github/ISSUE_TEMPLATE/2_bug.md:26]: i picked a "description" over "details" here because this template because it just better suits imo

Comment (TahaTesser) [reply .github/ISSUE_TEMPLATE/2_bug.md:26]: If we use "Details", most people would end up describing the nature of the issue rather than providing necessary steps to replicate the issue.

File: .github/ISSUE_TEMPLATE/2_bug.md (hunk 3 above)
Comment (TahaTesser) [reply .github/ISSUE_TEMPLATE/2_bug.md:30]: `flutter create ...` is necessary so the reporter is able to reproduce the issue outside their production app which would make the issue more likely related to Flutter than third-party packages, issues with their code and other reasons. We've many issues reported where authors follow these steps and provide reproducible from scratch `flutter create..`, modifying `main.dart`,  making changes to `pubspec.yaml` to running the app, which leaves the production app or existing app out as cause the of the issue.

Comment (nt4f04uNd) [reply .github/ISSUE_TEMPLATE/2_bug.md:30]: if we want to keep concrete instructions how do you create a minimal reproducible, we would like to rephrase this to express them as intructions for submitter, not something he will likely leave as an actual reproductrion step

i.e. that should be something that goes after this paragraph https://github.com/flutter/flutter/pull/79594/files#diff-a5bf860cf261d12c428dcd1164243e6a454f770d07b2b9b4e933d6f76d3051f9R50-R51

File: .github/ISSUE_TEMPLATE/2_bug.md (hunk 8 above)
Comment (nt4f04uNd) [reply .github/ISSUE_TEMPLATE/2_bug.md:26]: steps to reproduce is part of "details"/"description" section (see below)

**Comment by nt4f04uNd** [comment]
@Hixie could you review it once more

File: .github/ISSUE_TEMPLATE/1_activation.md (hunk 2 above)
Comment (Hixie) [reply .github/ISSUE_TEMPLATE/1_activation.md:46]: mostly the advantage of saying --verbose is that it's self-documenting. But I don't feel strongly about it.

File: .github/ISSUE_TEMPLATE/2_bug.md (hunk 8 above)
Comment (Hixie) [reply .github/ISSUE_TEMPLATE/2_bug.md:26]: literally the only thing we're asking for here is the steps to reproduce though, no?

Comment (Hixie) [reply .github/ISSUE_TEMPLATE/2_bug.md:26]: in this section i mean

**Comment by Hixie** [comment]
I feel like there's a lot of changes here, some of which seem valuable, some of which are cosmetic, and some of which are debatable, and it's hard for me to split these apart. Might it make sense to do this as several PRs instead? That way we can land the clear improvements that are fixing problems you're running into, and then we can debate the cosmetic and other suggestions independent of each other.

**Comment by nt4f04uNd** [comment]
i will consider doing this
